spec/requests/api/picture_spec.rb

```diff
@@ -63,26 +67,10 @@ def expect_result_to_include_picture_href(source_id)
   end
 
   describe 'POST /api/pictures' do
-    # one pixel png image encoded in Base64
```

Comment:
@imtayadeway `strict_decode64` had some issues with all of the newlines. Apparently it might not have been base64 😳